django/db/models/fields/files.py

```diff
@@ -173,9 +171,7 @@ def __get__(self, instance, cls=None):
 
         # The instance dict contains whatever was originally assigned
         # in __set__.
-        if self.field.name not in instance.__dict__:
-            instance.refresh_from_db(fields=[self.field.name])
-        file = instance.__dict__[self.field.name]
+        file = super().__get__(instance, cls)
```

Comment:
You can remove also lines [161-163](https://github.com/django/django/pull/13052/files#diff-77d4793905e702bfe9a8fabac708dcc7L161-L163).

Reply:
But in this case, if the attribute is accessed from the class (model_cls.attribute), then `DeferredAttribute .__ get __()` will return a FileDescriptor instance, but not the reloaded value.

What do you think?

Reply:
Yes, agreed. Let's leave these lines. Changes in descriptors are always tricky, I will try to review it properly in the next week.

Reply:
Ok. Thank you.